app/routers/profile.py

```diff
@@ -41,14 +42,12 @@ async def profile(
         request: Request,
         session=Depends(get_db),
         new_user=Depends(get_placeholder_user)):
-    # Get relevant data from database
-    upcoming_events = range(5)
     user = session.query(User).filter_by(id=1).first()
     if not user:
         session.add(new_user)
         session.commit()
         user = session.query(User).filter_by(id=1).first()
-
+    upcoming_events = get_upcoming_events(session, user.id)[:5]
```

Comment:
what is 5?

Reply:
I added a constant variable and an explanation to it.
We are presenting up to five upcoming events on the profile page